agent/tool-scripts/postprocess/watches-cluster-health-postprocess

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/perl
+
+# Author: Lukas Vlcek (lvlcek@redhat.com)
+#
+# usage: <tool>-postprocess <dir>  dir = directory where tool txt file can be found
+#
+# The purpose of this script is to
+# -output the chart data in JSON format (watches.js)
```

Comment:
I took this comment from other postprocessing script (I think it was `pidstat-postprocess`, but similar _formula_ can be found in may other scripts). May be we should open more general ticket for this?